core/src/main/java/com/rudderstack/android/sdk/core/RudderDeviceModeManager.java

```diff
@@ -200,7 +213,7 @@ void dumpEventToDestinations(RudderMessage message, List<String> destinations, S
             RudderIntegration<?> integration = integrationOperationsMap.get(destinationName);
             if (integration != null) {
                 try {
-                    RudderLogger.logDebug(String.format(Locale.US, "RudderDeviceModeManager: %s: dumping for %s", logTag, destinationName));
+                    RudderLogger.logDebug(String.format(Locale.US, "RudderDeviceModeManager: %s: dumping for %s and message: %s", logTag, destinationName, gson.toJson(message)));
```

Comment:
Let's not print the message directly, this will create more noise in the logs, and also when we are printing logs in this detail, it would be better to use verbose

Reply:
So does this look fine.
`RudderLogger.logDebug(String.format(Locale.US, "RudderDeviceModeManager: Dumping %s", gson.toJson(message)));`
Shall I proceed with it!

Reply:
Yeah this is fine